common/events.cpp

```diff
@@ -156,10 +156,12 @@ static struct {
     GEN_EV(E_TREATY_PEACE, E_S_TREATY, N_("Peace")),
     GEN_EV(E_TREATY_SHARED_VISION, E_S_TREATY, N_("Shared Vision")),
     GEN_EV(E_UNIT_LOST_ATT, E_S_UNIT, N_("Attack Failed")),
+    GEN_EV(E_UNIT_TIE_ATT, E_S_UNIT, N_("Attack Tied")),
     GEN_EV(E_UNIT_WIN_ATT, E_S_UNIT, N_("Attack Succeeded")),
     GEN_EV(E_UNIT_BUY, E_S_UNIT, N_("Bought")),
     GEN_EV(E_UNIT_BUILT, E_S_UNIT, N_("Built")),
     GEN_EV(E_UNIT_LOST_DEF, E_S_UNIT, N_("Defender Destroyed")),
+    GEN_EV(E_UNIT_TIE_DEF, E_S_UNIT, N_("Defender Tied")),
```

Comment:
Is the original table sorted in the same way as the list in `events.h`? If so, keep the same order for both.

Reply:
they are not in the same order as events.h